coordinator/manifest/manifest_test.go

```diff
@@ -16,7 +16,7 @@ import (
 
 	"github.com/edgelesssys/marblerun/coordinator/quote"
 	"github.com/edgelesssys/marblerun/coordinator/user"
-	"github.com/edgelesssys/marblerun/test"
+	test "github.com/edgelesssys/marblerun/test"
```

Comment:
```suggestion
	"github.com/edgelesssys/marblerun/test"
```
this also appears in a few other files